packages/core/src/external/carequality/domain/ihe-result.ts

```diff
@@ -0,0 +1,54 @@
+export interface BaseResultDomain {
+  requestId: string;
+  status: string;
+  createdAt: Date;
+}
+
+export type Issue = {
+  severity: string;
+  code: string;
+  details: { text: string };
+};
+
+export type OperationOutcome = {
+  resourceType: string;
+  id: string;
+  issue: Issue[];
+};
+
+export type BaseResponse = {
+  id: string;
+  cxId: string;
+  timestamp: string;
+  responseTimestamp: string;
+  xcpdPatientId?: { id: string; system: string };
+  patientId: string;
+  operationOutcome?: OperationOutcome;
+};
+
+export type DocumentReference = {
+  homeCommunityId: string;
```

Comment:
why redefining? they already exist in ihe-gateway-sdk. Ditto for OperationOutcome, Issue, etc